src/emqx_acl_ldap.erl

```diff
@@ -32,6 +32,9 @@
 
 -import(emqx_auth_ldap_cli, [search/3]).
 
+-import(emqx_auth_ldap, [replace_vars/2,
```

Comment:
I would rather reduce usage of import statements as they make code harder to read.

Reply:
Will change after the current implementation is verified.